File: src/main/java/com/ftn/domzdravlja/service/OsobljeService.java
package com.ftn.domzdravlja.service;

import java.util.List;

import com.ftn.domzdravlja.model.Osoblje;

public interface OsobljeService {

	Osoblje findOsobljeById(Integer id);
	
	Osoblje save(Osoblje osoblje);
	
	List<Osoblje> getOsobljeByKlinika(Integer klinikaId);
}

File: src/main/resources/my-app/src/services/OsobljeService.js
import axios from 'axios'

const USERS_REST_API_URL = "/domZdravlja/osoblje";

class OsobljeService {
    getOsoblje() {
        return axios.get(USERS_REST_API_URL);
    }  
    getOsobljeById(id){
        return axios.get(`${USERS_REST_API_URL}/${id}`);
    }

    getOsobljeByKlinika(klinikaId){
        return axios.get(`${USERS_REST_API_URL}/klinika/${klinikaId}`)
    }
}
export default new OsobljeService();